### ISSUE TITLE
When running steps in parallel, a task should be queued

### ISSUE THREAD

**Issue by fivedots** [whatwg/fs#36]
This issue was originally discussed here: https://github.com/whatwg/fs/pull/21/files#r879343956

The spec in general runs steps in parallel without enqueuing a task. It should be updated as mentioned in the comment chain above to use the right approach.

**Comment by a-sully** [comment]
Closing in favor of #3
